packages/twenty-front/src/modules/ui/navigation/menu-item/components/MenuItem.tsx

```diff
@@ -63,14 +65,18 @@ export const MenuItem = ({
       <StyledMenuItemLeftContent>
         <MenuItemLeftContent LeftIcon={LeftIcon ?? undefined} text={text} />
       </StyledMenuItemLeftContent>
-      <div className="hoverable-buttons">
-        {showIconButtons && (
-          <LightIconButtonGroup iconButtons={iconButtons} size="small" />
-        )}
-      </div>
       <StyledMenuItemRightContent>
-        <MenuItemLeftContent LeftIcon={RightIcon ?? undefined} text={''} />
+        {hasRightIcon && (
+          <MenuItemLeftContent LeftIcon={RightIcon ?? undefined} />
```

Comment:
LeftIcon={RightIcon ?

Reply:
Oh yes, I wanted to ask you about it, as I saw that there is a MenuItemLeftContent but not a MenuItemRightContent. Should I create MenuItemRightContent? I though that adding a RightIcon prop to the MenuItemLeftContent isn't a good practice either. What is preferable? I'm sorry for the mediocre code quality, trying my best as I'm still learning :)

Reply:
@charlesBochet check Discord :)